examples/dlt/README.md

```diff
@@ -29,6 +29,12 @@ It includes a pipeline to ingest messages from Slack channels and generate threa
     python run.py general dlt
     ```
 
+Or run it in Google Colab:
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)
+](https://colab.research.google.com/github/dagworks-inc/hamilton/blob/main/examples/dlt/notebook.ipynb)
```

Comment:
This one won't work because you need to do CLI commands like `dlt init` to get setup and the notebook requires access to local `slack/` subdirectory